### ISSUE TITLE
gistic2: Installed files are writable by group and some by other(!)

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/CBI-software#50]
```
$ ls -la /home/hb/shared/software/CBI/gistic2-2.0.23
total 7860
drwxrwx---   9 hb hb    4096 Mar 15 21:48 .
drwxr-xr-x 123 hb hb    4096 Mar 15 21:48 ..
drwxrwxr-x   2 hb hb    4096 Mar 27  2017 examplefiles
drwxrwxr-x   2 hb hb    4096 Mar 27  2017 example_results
-rwxrwxr-x   1 hb hb     709 Mar 15 21:48 gistic2
-rwxrwxr-x   1 hb hb     657 Mar 15 21:48 gistic2.org
drwxrwxr-x   2 hb hb    4096 Mar 27  2017 GISTICDocumentation_standalone_files
-rwxrwxr-x   1 hb hb   20348 Apr 14  2020 GISTICDocumentation_standalone.htm
-rwxrwxr-x   1 hb hb 7938873 Mar 27  2017 gp_gistic2_from_seg
-rwxrwxr-x   1 hb hb    2488 Mar 27  2017 INSTALL.txt
-rwxrwxr-x   1 hb hb    9464 Mar 27  2017 LICENSE.txt
drwxrwxr-x   3 hb hb    4096 Mar 15 21:48 MATLAB_Compiler_Runtime
drwxrwxr-x   7 hb hb    4096 Mar 15 21:48 MCR_Installer
-rwxrwxr-x   1 hb hb   11179 Mar 27  2017 README.txt
drwxrwxr-x   2 hb hb    4096 Mar 27  2017 refgenefiles
-rwxrwxr-x   1 hb hb     683 Mar 27  2017 run_gistic_example
drwxrwxr-x   4 hb hb   12288 Mar 27  2017 source
```

**Comment by HenrikBengtsson** [comment]
```sh
$ find -type f -perm -o=w
./gistic2-2.0.23/MCR_Installer/archives/matlab_graphics_java_common_1391823257.xml
./gistic2-2.0.23/MCR_Installer/archives/RptgenSL_resources_common_1388183753.xml
./gistic2-2.0.23/MCR_Installer/archives/ws_clients_common_1388183359.xml
./gistic2-2.0.23/MCR_Installer/archives/xmlxrcsc_glnxa64_1391007890.xml
...
./gistic2-2.0.23/MCR_Installer/archives/polyspace/cxx_headers_common_1388183303.xml
./gistic2-2.0.23/MCR_Installer/archives/glue2_resources_common_1389320919.xml
./gistic2-2.0.23/MCR_Installer/archives/map_resources_glnxa64_1391013090.xml
./gistic2-2.0.23/MCR_Installer/archives/parallel_gpu_libs_glnxa64_1391020690.xml
./gistic2-2.0.23/MCR_Installer/archives/services_glnxa64_1392179860.xml
```

**Comment by HenrikBengtsson** [comment]
This was fixed by #47
